README.md

```diff
@@ -578,6 +578,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Countly](http://resources.count.ly/docs) | Countly web analytics | No | No | Unknown |
 | [Drupal.org](https://www.drupal.org/drupalorg/docs/api) | Drupal.org | No | Yes | Unknown |
+| [Evil Insult Generator](https://evilinsult.com/api) | Open Source API for hand curated & evil insults | No | Yes | Yes |
```

Comment:
Can you please remove "API" from the description? "Evil insults" would fit better here.